Source/WebCore/rendering/style/RenderStyleInlines.h

```diff
@@ -812,7 +812,7 @@ inline bool RenderStyle::useTouchOverflowScrolling() const { return m_rareInheri
 #endif
 
 #if ENABLE(TEXT_AUTOSIZING)
-inline Length RenderStyle::initialSpecifiedLineHeight() { return { -100.0f, LengthType::Percent }; }
+inline Length RenderStyle::initialSpecifiedLineHeight() { return { -100.0f, LengthType::Normal }; }
```

Comment:
```suggestion
inline Length RenderStyle::initialSpecifiedLineHeight() { return LengthType::Normal; }
```

This should just work.